src/debug-adapter/debugEventManager.ts

```diff
@@ -151,7 +151,10 @@ export class DebugEventManager {
     }
 
     if (isNewAndObject && variable.variablesReference) {
-      const childVariables = await this.debugSessionProxy?.getVariables(variable.variablesReference);
+      let childVariables = await this.debugSessionProxy?.getVariables(variable.variablesReference);
+      if (variable.presentationHint?.lazy && childVariables && childVariables.length > 0) {
```

Comment:
We could write
```suggestion
      if (variable.presentationHint?.lazy && childVariables?.length > 0) {
```
here, if conciseness is preferred, but 
```typescript
      if (variable.presentationHint?.lazy && childVariables && childVariables.length > 0) {
```
is easy to understand.

Reply:
I had that idea as well but VS Code doesn't understand that for some reason:
![image](https://user-images.githubusercontent.com/80830960/162945287-d3bb0b66-bea5-456b-bdb8-7b1895d0dfdc.png)